core/src/test/java/io/parsingdata/metal/data/ByNameTest.java

```diff
@@ -25,7 +26,7 @@
 
     @Test
     public void getValueOnEmpty() {
-        assertNull(ByName.getValue(ParseGraph.EMPTY, "name"));
+        TestCase.assertNull(ByName.getValue(ParseGraph.EMPTY, "name"));
```

Comment:
Not sure what happened here, but use a static import as before. Also: I think we shouldn't use `TestCase.*` but `Assert.*`.